CHANGELOG.md

```diff
@@ -49,6 +49,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1512](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1512))
 - Add commit method for ConfluentKafkaInstrumentor's ProxiedConsumer
   ([#1656](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1656))
+- Fix confluent-kafka instrumentation by allowing Producer headers to be dict or list
+  ([#1655](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1655))
```

Comment:
This should be moved to Unreleased fixed section now.

Reply:
@srikanthccv I think that's where it is right now, if I view the markdown, it shows up in the Unreleased Fixed section: https://github.com/open-telemetry/opentelemetry-python-contrib/blob/739e1f0564b12e12788688d8de48adea3b3d7b2f/CHANGELOG.md#fixed

Reply:
There was a release on Friday, so the prev section is tagged and there is a new Unreleased section. Look at the diff.

Reply:
oh I hadn't rebased with main yet, that's why it was pointing to the older commit...updated it now and moved to the new unreleased section